web-ui/src/components/member-directory/AdminMemberCard.jsx

```diff
@@ -210,6 +215,10 @@ const AdminMemberCard = ({ member, index }) => {
               open={open}
               onClose={handleClose}
               onSave={async member => {
+                let resGetMember = await getMember(member.id, csrf);
+                let oldMember = resGetMember.payload?.data && !resGetMember.error
+                    ? resGetMember.payload.data
+                    : null;
```

Comment:
I noticed this component has `member` passed as a prop. Is that object different from what's returned by the payload of a successful `getMember` request?

Reply:
Yes member is the updated profile, while getMember is getting the old profile (if it exists). Its used to see if the pdl/supervisor has changed.

Reply:
Got it. Thank you. In that case, if you wanted to parallelize the two requests (getMember and updateMember), you could:

```js
const [resGetMember, res] = await Promise.all([
  getMember(member.id, csrf),
  updateMember(member, csrf),
]);
```

This could save 300ms or so by avoiding a non-dependent http request from waiting on the other to complete (as it does using `await` in sequence).

Reply:
If the update happens before the get. I wont be guaranteed to have the old member right?

Reply:
Oh I didn't realize they were dependent. That's right.